phoenix-core/src/main/java/org/apache/phoenix/iterate/RegionScannerFactory.java

```diff
@@ -109,16 +109,15 @@ public RegionScanner getWrappedScanner(final RegionCoprocessorEnvironment env,
       final Expression[] arrayFuncRefs, final int offset, final Scan scan,
       final ColumnReference[] dataColumns, final TupleProjector tupleProjector,
       final Region dataRegion, final IndexMaintainer indexMaintainer,
-      PhoenixTransactionContext tx,
-      final byte[][] viewConstants, final KeyValueSchema kvSchema,
+      PhoenixTransactionContext tx, final byte[][] viewConstants, final KeyValueSchema kvSchema,
       final ValueBitSet kvSchemaBitSet, final TupleProjector projector,
       final ImmutableBytesWritable ptr, final boolean useQualifierAsListIndex) {
     return new RegionScanner() {
-
       private RegionInfo regionInfo = env.getRegionInfo();
       private byte[] actualStartKey = getActualStartKey();
       private boolean useNewValueColumnQualifier = EncodedColumnsUtil.useNewValueColumnQualifier(scan);
 
+
```

Comment:
nit: seems like you can revert this file

Reply:
Done